src/test/regress/expected/multi_read_from_secondaries.out

```diff
@@ -24,10 +24,10 @@ INSERT INTO dest_table (a, b) VALUES (2, 1);
 INSERT INTO source_table (a, b) VALUES (10, 10);
 -- simluate actually having secondary nodes
 SELECT * FROM pg_dist_node;
- nodeid | groupid | nodename  | nodeport | noderack | hasmetadata | isactive | noderole | nodecluster 
---------+---------+-----------+----------+----------+-------------+----------+----------+-------------
-      1 |       1 | localhost |    57637 | default  | f           | t        | primary  | default
-      2 |       2 | localhost |    57638 | default  | f           | t        | primary  | default
+ nodeid | groupid | nodename  | nodeport | noderack | hasmetadata | isactive | noderole | nodecluster | isdatanode 
```

Comment:
this test needs to be updated anytime we change `pg_dist_node`, can we instead only select fields relevant for this test.